app/src/androidTest/java/com/github/polypoly/app/WelcomeActivityTest.kt

```diff
@@ -17,7 +21,7 @@ class WelcomeActivityTest {
     private val optionButtonRules = composeTestRule.onNodeWithContentDescription("Show Rules")
     private val optionButton2 = composeTestRule.onNodeWithContentDescription("optionButton2")
```

Comment:
UI component should have a more meaningful name and id to express their intent

Reply:
This works also for optionButton3

Reply:
Why are they call **Option**Button btw?

Reply:
I agree with you, the problem is that they don't have a fixed utility so far, I put some launching intents but only for the demo.
We need to decide what button does what, I'm gonna change that

Reply:
Great, this looks fine now that you added details regarding what option they refer to